incubator/jaeger/values.yaml

```diff
@@ -131,18 +179,64 @@ query:
   ## Additional pod labels
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
   podLabels: {}
+
+spark:
+  enabled: false
+  annotations: {}
+  image: jaegertracing/spark-dependencies
+  tag: latest
+  pullPolicy: Always
+  schedule: "*/12 * * * *"
```

Comment:
It should run just before the end of the day. Let say 5-10 min for startup

Reply:
I configured it to run every day @ 23:49